webapp/store/views.py

```diff
@@ -38,16 +38,14 @@
 ]
 
 CATEGORIES = [
-    {"slug": "ai/ml", "name": "AI/ML"},
+    {"slug": "ai-ml", "name": "AI/ML"},
     {"slug": "big-data", "name": "Big Data"},
-    {"slug": "database", "name": "Database"},
     {"slug": "cloud", "name": "Cloud"},
-    {"slug": "containers", "name": "Containers"},
-    {"slug": "featured", "name": "Featured"},
-    {"slug": "logging-and-tracing", "name": "Logging and Tracing"},
+    {"slug": "database", "name": "Database"},
+    {"slug": "kubernetes", "name": "Kubernetes"},
```

Comment:
was this new category agreed somewhere?

Reply:
ah I got my answer. I opened a bug on launchpad. we should keep what we had: https://bugs.launchpad.net/snapstore-server/+bug/1921382

Reply:
ahh, ok. I'll revert that part.